src/components/icons.js

```diff
@@ -60,14 +60,19 @@ export const centeredSpinner = ({ size = 48, ...props } = {}) => spinner(_.merge
   }
 }, props))
 
-export const profilePic = ({ size, style, ...props } = {}) => img({
-  alt: 'Google profile image',
-  src: getUser().imageUrl,
-  height: size, width: size,
-  style: { borderRadius: '100%', ...style },
-  referrerPolicy: 'no-referrer',
-  ...props
-})
+export const profilePic = ({ size, style, ...props } = {}) => {
+  // Note Azure logins don't currently have an imageUrl, so don't render anything.
+  // See TOAZ-147 to support this in the future.
```

Comment:
This now looks like this for the B2C-Azure case (note no picture):

<img width="282" alt="image" src="https://user-images.githubusercontent.com/5368863/168683533-964ca166-2199-4540-8df9-88551c5bfbc1.png">

Potential options for the future:
1. Autogenerate a pic if one isn't present. TDR UI [auto-generates](https://github.com/DataBiosphere/jade-data-repo-ui/blob/develop/src/components/common/TerraAvatar.tsx) one using [material-ui/react-avatar](https://mui.com/material-ui/react-avatar/) (cc @nmalfroy). I was hesitant to add this dependency to Terra UI however.

2. Render a static image like 
   <img width="50" alt="image" src="https://user-images.githubusercontent.com/5368863/168682914-91849963-ec38-4bd0-9ec9-5ada62af7fd5.png">

4. Or do something on the B2C side.